docs/docs/configuration/alerts-reports.mdx

```diff
@@ -246,25 +246,28 @@ USER superset
 ### Using Chrome
 
 ```docker
-FROM apache/superset:3.1.0
-
-USER root
+ARG SUPERSET_IMAGE=3.1.3
+FROM docker.io/apache/superset:${SUPERSET_IMAGE}
 
-RUN apt-get update && \
-    apt-get install -y wget zip libaio1
+ARG CHROME_VERSION=115.0.5765.0
 
-RUN export CHROMEDRIVER_VERSION=$(curl --silent https://googlechromelabs.github.io/chrome-for-testing/LATEST_RELEASE_116) && \
-    wget -O google-chrome-stable_current_amd64.deb -q http://dl.google.com/linux/chrome/deb/pool/main/g/google-chrome-stable/google-chrome-stable_${CHROMEDRIVER_VERSION}-1_amd64.deb && \
-    apt-get install -y --no-install-recommends ./google-chrome-stable_current_amd64.deb && \
-    rm -f google-chrome-stable_current_amd64.deb
-
-RUN export CHROMEDRIVER_VERSION=$(curl --silent https://googlechromelabs.github.io/chrome-for-testing/LATEST_RELEASE_116) && \
-    wget -q https://storage.googleapis.com/chrome-for-testing-public/${CHROMEDRIVER_VERSION}/linux64/chromedriver-linux64.zip && \
-    unzip -j chromedriver-linux64.zip -d /usr/bin && \
-    chmod 755 /usr/bin/chromedriver && \
-    rm -f chromedriver-linux64.zip
+USER root
 
-RUN pip install --no-cache gevent psycopg2 redis
+RUN apt-get update \
+    && apt-get install -y --no-install-recommends \
+        curl unzip jq \
+        libglib2.0-0 libnss3 libxcb1 libatk1.0-0 libatk-bridge2.0-0 \
+        libcups2 libdrm2 libxkbcommon0 libxcomposite1 libxdamage1 \
+        libxfixes3 libxrandr2 libgbm1 libpango-1.0-0 libcairo2 libasound2 \
+    && apt-get clean && rm -rf /var/lib/apt/lists/*
+
+RUN curl -s -O -O \
+    $(curl -s \
+        https://googlechromelabs.github.io/chrome-for-testing/known-good-versions-with-downloads.json \
```

Comment:
Looks like this will lead to non-deterministic builds (re-running the same docker build can lead to different binaries) which is not ideal. I'd like to keep our build as deterministic as possible.